assets/js/src/common/sender_email_address_warning.jsx

```diff
@@ -1,43 +1,111 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import MailPoet from 'mailpoet';
+import moment from 'moment';
 import ReactStringReplace from 'react-string-replace';
 
-const SenderEmailAddressWarning = ({ emailAddress }) => {
+const userHostDomain = window.location.hostname.replace('www.', '');
+const suggestedEmailAddress = `contact@${userHostDomain}`;
+
+const NewInstallationWithMSSWarning = ({ emailAddress }) => (
+  <React.Fragment>
+    <p
+      className="sender_email_address_warning"
+      data-acceptance-id="freemail-sender-warning-new-installation"
+    >
+      {ReactStringReplace(
+        MailPoet.I18n.t('senderEmailAddressNewInstallWarning1'),
+        '%originalSender',
+        () => <em key="reply-to">{ emailAddress }</em>
+      )}
+    </p>
+    <p className="sender_email_address_warning">
+      {ReactStringReplace(
+        MailPoet.I18n.t('senderEmailAddressNewInstallWarning2'),
+        /(%suggested|%originalSender)/,
+        (match) => {
+          if (match === '<em>%suggested</em>') return <em key="suggested-email">{ suggestedEmailAddress }</em>;
```

Comment:
This condition should be `match === '%suggested'`

Reply:
Thanks Amine, I totally missed that, there was a conflict I had to resolve.